packages/harmony/src/components/layout/Flex/Flex.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Meta, StoryObj } from '@storybook/react'
+
+import { Text } from '../../typography/Text'
+import { Box } from '../Box'
+
+import { Flex } from './Flex'
+
+const meta: Meta<typeof Flex> = {
+  title: 'Components/Layout/Flex',
+  component: Flex,
+  parameters: {
+    controls: { include: /^(alignItems|direction|gap|justifyContent|wrap)$/ }
+  },
+  args: {
+    gap: 'm'
+  },
+  argTypes: {
+    alignItems: {
+      control: { type: 'select' },
```

Comment:
does radio take up too much space? I think it's significantly easier to use

Reply:
Yea radio was taking up a lot of space so I opted to use select on props that had more than 3 options. Could change back on flex, but for something like Box it was a lot of scrolling bc there were so many padding and margin props